GNUmakefile

```diff
@@ -451,5 +451,11 @@ gnulib-update:
 	  --source-base=lib/gnu --lgpl=2 --no-vc-files --no-conditional-dependencies \
 	  $(GNU_MODS_AVOID:%=--avoid %) --import $(GNU_MODS)
 
-ancillary-clean: spec-clean srpm-clean mock-clean coverity-clean
+## Coverage/profiling
+
+.PHONY: coverage-clean
+coverage-clean:
+	-find . \( -name "*.gcno" -o -name "*.gcda" \) -exec rm -f \{\} \;
+
+ancillary-clean: spec-clean srpm-clean mock-clean coverity-clean coverage-clean
```

Comment:
I'm leaning to keeping this as-is for convenience, but I wonder if technically ancillary-clean should be done with maintainer-clean instead of clean